src/components/KMultiselect/KMultiselect.vue

```diff
@@ -604,10 +604,19 @@ const stageSelections = () => {
     if (elem) {
       const height = elem.clientHeight
       if (height > selectionsMaxHeight.value) {
-        const item = visibleSelectedItemsStaging.value.pop()
-        if (item && !invisibleSelectedItemsStagingSet.has(item.value)) {
-          invisibleSelectedItemsStagingSet.add(item.value)
-          invisibleSelectedItemsStaging.value.push(item)
+        // populate as much items as possible by checking the offsetTop
+        const overflowedElements = Array.from(elem.querySelectorAll('.k-multiselect-selection-badge'))
+          .filter($badge => ($badge as HTMLElement).offsetTop >= selectionsMaxHeight.value)
```

Comment:
Do we want to remove the `$` prefix here as well?
```suggestion
          .filter(badge => (badge as HTMLElement).offsetTop >= selectionsMaxHeight.value)
```

Reply:
Ooooops